trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/set-record-correction-status.ts

```diff
@@ -1,34 +1,27 @@
-/**
- * returns the record correction status string
- * based on if record correction is completed or not
- * @param isCompleted - if record correction is completed
- * @returns 'Record correction sent' or 'Record correction received'
- */
-export const setRecordCorrectionDisplayStatus = (isCompleted: boolean) => {
-  const recordCorrectionSent = 'Record correction sent';
-  const recordCorrectionReceived = 'Record correction received';
-
-  if (isCompleted) {
-    return recordCorrectionReceived;
-  }
-
-  return recordCorrectionSent;
-};
+import { RecordCorrectionStatus, RecordCorrectionDisplayStatus } from '../../../types/view-models';
 
 /**
- * sets the record correction status based on isCompleted
- * if true, then returns RECORD_CORRECTION_RECEIVED
- * else returns RECORD_CORRECTION_SENT
+ * returns displayStatus and status
+ * based on if record correction is completed or not
  * @param isCompleted - if record correction is completed
- * @returns RECORD_CORRECTION_RECEIVED if completed else RECORD_CORRECTION_SENT
+ * @returns displayStatus and status
  */
 export const setRecordCorrectionStatus = (isCompleted: boolean) => {
```

Comment:
Please could we rename this to `getRecordCorrectionStatus` or `mapToRecordCorrectionStatus`, the word 'set' implies to me we are actually changing an existing object, which is not what is happening here

Reply:
renamed